test/appium/tests/atomic/chats/test_one_to_one.py

```diff
@@ -1,131 +1,412 @@
+        self.device_1.profile_button.click()
+        self.device_1.click_system_home_button()
+
+        self.device_2.just_fyi("Check text push notification and tap it")
+        self.device_2.open_notification_bar()
+        if not self.home_2.get_pn(message):
+            self.device_2.driver.fail("Push notification with text was not received")
+        chat_2 = self.device_2.click_upon_push_notification_by_text(message)
+
+        self.device_2.just_fyi("Send emoji message to Device 1 while it's on backround")
         emoji_message = random.choice(list(emoji.EMOJI_UNICODE))
         emoji_unicode = emoji.EMOJI_UNICODE[emoji_message]
         chat_2.send_message(emoji.emojize(emoji_message))
 
-        device_1.just_fyi("Device 1 checks PN with emoji")
-        device_1.open_notification_bar()
-        if not device_1.element_by_text_part(emoji_unicode).is_element_displayed(10):
-            device_1.driver.fail("Push notification with emoji was not received")
-        chat_1 = device_1.click_upon_push_notification_by_text(emoji_unicode)
+        self.device_1.just_fyi("Device 1 checks PN with emoji")
+        self.device_1.open_notification_bar()
+        if not self.device_1.element_by_text_part(emoji_unicode).is_element_displayed(10):
+            self.device_1.driver.fail("Push notification with emoji was not received")
+        chat_1 = self.device_1.click_upon_push_notification_by_text(emoji_unicode)
 
-        device_1.just_fyi("Check Device 1 is actually on chat")
+        self.device_1.just_fyi("Check Device 1 is actually on chat")
         if not (chat_1.element_by_text_part(message).is_element_displayed()
                 and chat_1.element_by_text_part(emoji_unicode).is_element_displayed()):
-            device_1.driver.fail("Failed to open chat view after tap on PN")
+            self.device_1.driver.fail("Failed to open chat view after tap on PN")
 
-        device_1.just_fyi("Checks there are no PN after message was seen")
-        [device.click_system_home_button() for device in (device_1, device_2)]
-        [device.open_notification_bar() for device in (device_1, device_2)]
-        if (device_2.element_by_text_part(message).is_element_displayed()
-                or device_1.element_by_text_part(emoji_unicode).is_element_displayed()):
+        self.device_1.just_fyi("Checks there are no PN after message was seen")
+        [device.click_system_home_button() for device in (self.device_1, self.device_2)]
+        [device.open_notification_bar() for device in (self.device_1, self.device_2)]
+        if (self.device_2.element_by_text_part(message).is_element_displayed()
+                or self.device_1.element_by_text_part(emoji_unicode).is_element_displayed()):
             self.errors.append("PN are keep staying after message was seen by user")
         self.errors.verify_no_errors()
 
+    @marks.testrail_id(6305)
+    @marks.critical
+    def test_image_in_one_to_one_send_save_reply_timeline(self):
```

Comment:
IMO better to remove from here check for timeline and have it separately (as far as I understand it is not a prio for now, so it is fine to have them in medium, i.e. as a part of test_different_status_in_timeline)

Reply:
Done